modAionImpl/src/org/aion/zero/impl/forks/ForkUtility.java

```diff
@@ -54,6 +54,16 @@ public ForkUtility(Properties properties, Logger log) {
 
             enableNonceFork(maybeNonceFork.get());
         }
+
+        Optional<Long> maybeSignatureSwapFork = loadSignatureSwapForkNumberFromConfig(properties);
+        if (maybeSignatureSwapFork.isPresent()) {
+            if (maybeSignatureSwapFork.get() < 2) {   // AKI-419, Constrain the minimum unity fork number
+                log.warn("The nonce fork number cannot be less than 2, set the fork number to 2");
```

Comment:
the warning should say fork1.7; also the comment regarding the AKI can be removed